src/api/seancesService.ts

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+import type { Seance } from '@/types/seance'
+
+axios.defaults.baseURL = 'http://localhost:3000';
+
+export const getAllSeance = async (): Promise<Seance[]> => {
+  const response = await axios.get('/seances') 
+  return response.data
+}
+
+export const getDaySeance = async (date: string): Promise<Seance[]> => {
+  const response = await axios.get(`/seances?date=${date}`) 
+  if (response) return response.data
```

Comment:
I believe response will be always defined but response body might be not.
```suggestion
  if (response.data) return response.data
```